core/src/main/java/org/jruby/java/dispatch/CallableSelector.java

```diff
@@ -700,7 +701,7 @@ private static int calcTypePreference(Class<?> type, final IRubyObject arg) {
                 return 1;
             }
         }
-        else if ( arg instanceof RubyString ) {
+        else if ( arg instanceof RubyString || arg instanceof RubySymbol ) {
```

Comment:
This is not a good idea, we should make a separate if case and only handle Java String or maybe CharSequence for RubySymbol.

we do not want to treat it exactly as a RubyString which could convert to a byte sequence.